js/app.js

```diff
@@ -0,0 +1,164 @@
+requestResourceButton.addEventListener('click', dataMaker);
```

Comment:
You should declare `requestResourceButton` and `resourceType` somewhere in this file.